system_notification/infra/http/server/flask_http_server.py

```
@@ -0,0 +1,71 @@
+    g = functools.update_wrapper(g, f)
+    g.__kwdefaults__ = f.__kwdefaults__
+    return g
+
+
+class FlaskHttpServer:
+    def __init__(self, debug: bool = False) -> None:
+        self._debug_mode = debug
+        self._app = Flask(__name__)
+
+    def serve(self, port: int = 8000) -> None:
+        self._app.run(host="0.0.0.0", port=port, debug=self._debug_mode)
```

Ruff (0.13.3)

Possible binding to all interfaces

(S104)